_posts/2023-02-15-my-first-patch-set.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title:  "My First Patch Set"
+date:   2023-02-15 11:57:26 -0300
+categories: jekyll update
+---
+
+Hello everyone! In this Post i will describe my experience writing my first patch set to the Linux
+Kernel, more specifically in the [Direct Rendering Manager (DRM)][wikipedia-drm] subsystem.
+
+## Why I'm Doing This
+
+Besides the fun of contributing to a project that millions of people use, This is part of my
+initiation on the [GSoC project Idea about Increasing Code Coverage on the DRM
+code][xorg-gsoc-project-idea], a project that I pretend to do on this year.
+
+## Patch Set Idea
+
+The idea is to decrease number of warnings generated when compiling the AMD GPU drivers with `W=1`.
+This is a pretty simple task to start working on, as is not needed an specific hardware to test it.
+
+Sometimes this work could be viewed as not useful, but it is a much needed job that sometimes is put
+aside on behalf of more urgent problems. It makes the code less error prone, more readable and
+simpler, helping others on developing.
+
+## Set Up
+
+To start I added the `drm-misc` tree as it was recommend.
```

Comment:
You can mention that it is also possible to use the amd-staging-drm-next branch.

Reply:
I myself did not know about that huahuaha, but I'm sure to put that